targets/xfce-desktop

```diff
@@ -7,8 +7,25 @@ DESCRIPTION='Installs Xfce along with common applications. (Approx. 1200MB)'
 . "${TARGETSDIR:="$PWD"}/common"
 
 ### Append to prepare.sh:
-# Avoid /etc/skel/.xscreensaver conflicts
-xsvr='/etc/skel/.xscreensaver'
-[ -f "$xsvr" ] && mv -f "$xsvr" "$xsvr.bak"
-install ubuntu=xubuntu-desktop,task-xfce-desktop -- network-manager xorg
-[ -f "$xsvr.bak" ] && mv -f "$xsvr.bak" "$xsvr"
+
+if [ "${DISTROAKA:-"$DISTRO"}" = 'gentoo' ]; then
+    # Use this target to install the rest of the xfce4 packages and more
+    install orage mousepad ristretto squeeze thunar-archive-plugin \
```

Comment:
An overlay isn't any better...I'm just worried that among a long list of packages, one may go missing.  But if the alternative is to place that long list elsewhere, it's preferable to keep it here.